DEV_3/DEV_3/Employee.cs

```diff
@@ -0,0 +1,250 @@
+        {
+            StringBuilder outputString = new StringBuilder();
+            outputString.AppendFormat(this.GetName);
+            outputString.Append(" ");
+            outputString.AppendFormat(this.GetType().ToString().Substring(this.GetType().ToString().LastIndexOf('.')));
+            return outputString.ToString();
+        }
+    }
+
+    /// <summary>
+    /// The junior. 
+    /// </summary>
+    public class Junior : Employee
+    {
+        /// <summary>
+        /// The salary.
+        /// </summary>
+        protected int salary;
+
+        /// <summary>
+        /// The productivity.
+        /// </summary>
+        protected int productivity;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Junior"/> class.
+        /// </summary>
+        public Junior()
+        {
+            this.salary = 50;
+            this.productivity = 15;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Junior"/> class.
+        /// </summary>
+        /// <param name="firstName">
+        /// The first name.
+        /// </param>
+        /// <param name="secondName">
+        /// The second name.
+        /// </param>
+        public Junior(string firstName = "test", string secondName = "test")
+            : base(firstName, secondName)
+        {
+            this.salary = 50;
+            this.productivity = 15;
+        }
+
+        /// <summary>
+        /// Gets the salary.
+        /// </summary>
+        public int GetSalary
+        {
+            get
+            {
+                return this.salary;
+            }
+        }
+
+        /// <summary>
+        /// Gets the efficiency coefficient.
+        /// </summary>
+        public double GetEfficiencyCoefficient
+        {
+            get
+            {
+                return this.productivity / (double)this.salary;
+            }
+        }
+
+        /// <summary>
+        /// Gets the productivity.
+        /// </summary>
+        public int GetProductivity
+        {
+            get
+            {
+                return this.productivity;
+            }
+        }
+
+        /// <summary>
+        /// All <see cref="Junior"/> field information
+        /// </summary>
+        /// <returns>Format string with <see cref="Junior"/> information</returns>
+        public override string GetInfo()
+        {
+            StringBuilder outputString = new StringBuilder();
+            outputString.AppendFormat(this.GetName);
+            outputString.Append(" ");
+            outputString.AppendFormat(this.GetType().ToString().Substring(this.GetType().ToString().LastIndexOf('.')));
+            outputString.AppendFormat(" Productivity:{0} , Salary:{1}", this.GetProductivity, this.GetSalary);
+            return outputString.ToString();
+        }
+    }
+
+    /// <inheritdoc />
+    /// <summary>
+    /// The middle.
+    /// </summary>
+    public class Middle : Junior
```

Comment:
Source files should contain one class definition per source file. Said differently, each class
definition will exist within its own file.